kernel/src/task/tasks.rs

```diff
@@ -267,6 +278,10 @@ impl Task {
         self.id
     }
 
+    pub fn get_rootdir(&self) -> Arc<dyn Directory> {
```

Comment:
```suggestion
    pub fn rootdir(&self) -> Arc<dyn Directory> {
```

In Rust, [getters don't have a `get_` prefix](https://rust-lang.github.io/api-guidelines/naming.html?highlight=gett#getter-names-follow-rust-convention-c-getter).